pkg/operator/k8sutil/cmdreporter/cmdreporter.go

```diff
@@ -333,11 +333,14 @@ func (cr *cmdReporterCfg) initContainers() []v1.Container {
 	}
 
 	c := v1.Container{
-		Name: CopyBinariesInitContainerName,
-		// Command: the rook command is the default entrypoint of rook images already
+		Name:    CopyBinariesInitContainerName,
+		Command: []string{"cp"},
 		Args: []string{
-			"copy-binaries",
-			"--copy-to-dir", CopyBinariesMountDir,
+			"--archive",
+			"--force",
+			"--verbose",
```

Comment:
How many lines does the verbose flag cause on the `cp` command in comparison to the current copy binaries command?

Reply:
@Sheetalpamecha How verbose is this flag? Do you have test output from the container?

Reply:
> How many lines does the verbose flag cause on the `cp` command in comparison to the current copy binaries command?

@galexrt Both ways container prints one line only. 
@travisn Here are both the outputs - 

with copy binaries cmd - 

```
sheetalpamecha@Sheetals ~/rook (init-copy-binaries)> kubectl -n rook-ceph logs rook-ceph-osd-prepare-minikube-n884d -c copy-bins
2023-07-31 04:02:15.507266 I | job-reporter-cmd: copying /usr/local/bin/rook to /rook/rook
```

with cp cmd -
```
runner@fv-az625-580:~/work/rook/rook$ k logs rook-ceph-osd-prepare-fv-az625-580-b6564 -c copy-bins
'/usr/local/bin/rook' -> '/rook/rook'
```
If we remove verbose flag nothing is printed in logs.